app/routes/events/components/EventEditor/index.tsx

```diff
@@ -125,10 +147,91 @@ function EventEditor({
           name="cover"
           component={ImageUploadField}
           uploadFile={uploadFile}
-          edit={isEditPage && ((token) => setCoverPhoto(eventId, token))}
           aspectRatio={20 / 6}
-          img={event.cover}
+          img={useImageGallery ? imageGalleryUrl : event.cover}
         />
+        <Modal
+          show={showImageGallery}
+          onHide={() => setShowImageGallery(false)}
+          contentClassName={styles.imageGallery}
+        >
+          <>
+            <Flex column>
+              <h1>Bildegalleri</h1>
+              <Flex wrap alignItems="center" justifyContent="space-around">
+                {imageGallery &&
+                  imageGallery.map((e) => {
+                    return (
+                      <div key={e.key} className={styles.imageGalleryContainer}>
+                        <ConfirmModalWithParent
+                          title="Fjern fra galleri"
+                          message="Er du sikker på at du vil fjerne bildet fra bildegalleriet? (Bildet blir ikke slettet!)"
+                          onConfirm={() =>
+                            setSaveForUse(`${e.key}:${e.token}`, false)
+                          }
+                          closeOnConfirm
+                        >
+                          <button className={styles.closeButton}>
+                            <Icon name="close-circle" size={24} />
+                          </button>
```

Comment:
You can move the onclick function from ConfirmModal into the Icon component and it will resemble all other icons instead of a custom closeButton style